ci/pipeline.yml

```diff
@@ -143,17 +145,39 @@ jobs:
         resource: develop-version
         params: { pre: beta }
 
-  - name: integration-develop
+  - name: continuously-integrate-develop
     serial: true
     plan:
       - aggregate:
         - get: repo
           resource: repo-develop
           trigger: true
-          passed: [ ship-develop ]
+          passed: [ continuously-ship-develop ]
       - task: integration-test
         file: repo/ci/tasks/integration.yml
 
+  - name: manually-ship-develop
```

Comment:
@mumoshu  If we put 'auto' for continuous job, I think 'manually' would be somewhat redundant.  I propose

- 'auto-ship|integrate-...' for continuous shipment and integrate jobs
- 'ship|integrate-...' for manual shipment and integrate jobs

How does this sound for you?